sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -39,6 +40,15 @@ def test_mcd():
     # 1D data set
     launch_mcd_on_dataset(500, 1, 100, 0.001, 0.001, 350)
 
+def test_fast_mcd():
+
```

Comment:
To be consistent with the style of the other functions in this file and the project in general, please remove this blank line.